frontend/packages/pipelines-plugin/src/types/coreTekton.ts

```diff
@@ -11,3 +11,14 @@ export type TektonResource = {
   optional?: boolean;
   type: string; // TODO: limit to known strings
 };
+
+export type TektonResultsDefinition = {
+  name: string;
+  description?: string;
+  value: string;
```

Comment:
I think value is optional 🤔 

It's like as if you were to craft the response vs just allowing it to be whatever is jammed into it during execution.

Reply:
Just confirmed only description is optional. value must be there in definition.

Reply:
Your example in your description says otherwise:
```
apiVersion: tekton.dev/v1beta1
kind: Task
metadata:
  name: add-task
spec:
  params:
    - name: first
      description: the first operand
    - name: second
      description: the second operand
  results:
    - name: sum
      description: the sum of the first and second operand
  steps:
    - name: add
      image: alpine
      env:
        - name: OP1
          value: $(params.first)
        - name: OP2
          value: $(params.second)
      command: ["/bin/sh", "-c"]
      args:
        - echo -n $((${OP1}+${OP2})) | tee $(results.sum.path);`
```